test/unit/filtered_replication.coffee

```diff
@@ -0,0 +1,82 @@
+should = require 'should'
+sinon = require 'sinon'
+filteredReplication = require '../../src/filtered_replication'
+
+cozyUrl = 'http://localhost'
+deviceName = 'local-device'
+password = 'password'
+
+describe "filteredReplication", ->
+
+    describe "general", ->
+
+        it "possible to get filter name from device name.", ->
+
+            filteredReplication.getFilterName(deviceName).should.equal \
+                'filter-local-device-config'
+
+        it "possible to get filter doc id from device name.", ->
+
+            filteredReplication.getFilteredDocId(deviceName).should.equal \
+                '_design/filter-local-device-config'
+
+        it "create filtered doc", ->
+
+            config = file: true
+            doc = filteredReplication.getFilteredDoc deviceName, config
+
+            id = filteredReplication.getFilteredDocId deviceName
+            key = filteredReplication.getFilterName deviceName
+            value = filteredReplication.getFilteredFunction config
+
+            doc._id.should.be.equal id
+            doc.views.should.be.empty()
+            doc.filters.should.be.eql "#{key}": value
+
+    describe "filtered function", ->
+
+        it "create empty filter function.", ->
+
+            fn = eval('(' + filteredReplication.getFilteredFunction() + ')')
```

Comment:
You could use string interpolation as below: https://github.com/cozy/cozy-device-sdk/pull/5/files#diff-919da6254b54bfa4c3ee3795c6f4e982R59